commander/src/bootstrapping/commands/generator/enable.ts

```diff
@@ -13,43 +13,94 @@
  *
  */
 
-import { BaseForgingCommand } from '../base_forging';
+import { Flags } from '@oclif/core';
+import { BaseGeneratorCommand } from '../base_generator';
 
-export abstract class EnableCommand extends BaseForgingCommand {
-	static description = 'Enable forging for given validator address.';
+const isLessThanZero = (value: number | undefined | null): boolean =>
+	value === null || value === undefined || value < 0;
```

Comment:
I find it semantically incorrect to consider `null` and `undefined` to be less than zero.

Consider renaming the function to `isNonNegativeNumber()` and reversing the conditional. Then don't forget to update the usages of the function by adding negation in front.

https://en.m.wikipedia.org/wiki/Sign_(mathematics)#non-negative_and_non-positive

Reply:
`isPositiveNumber()` is a little bit better, but is still not 100% correct, because 0 is not a positive number 🤓 

There is a reason I suggested `isNonNegativeNumber()` instead of `isPositiveNumber()`

Reply:
yea that's true, but name wise nonNegativeNumber is not so straight forward. I think `isUInt` will probably be more suited

Reply:
`isUInt()` is definitely the perfect name for what we need there. Probably haven't thought of it before, because the function never checked that input number was an integer 🙂